transformer/test/transform/test_dataframe.py

```diff
@@ -105,42 +105,6 @@ def test_clean_data(_, mock_sha1, dict_map_gender, dict_map_code):
     assert cleaned_data == expected
 
 
-def test_squash_rows():
```

Comment:
would be nice to have a test to make sure group_by actually behaves as expected. We plan to use sqlite for "unit" tests in order to test sql queries behaviour (see #220). can you create an issue in order not to forget to test this ? ( or comment on #220)